coderd/database/migrations/000248_notifications_manual_build_failed.up.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO notification_templates (id, name, title_template, body_template, "group", actions)
+VALUES ('2faeee0f-26cb-4e96-821c-85ccb9f71513', 'Workspace Manual Build Failed', E'Workspace "{{.Labels.name}}" manual build failed',
+        E'Hi {{.UserName}},\nA manual build of the workspace **{{.Labels.name}}** using the template **{{.Labels.template_name}}** failed.\nThe workspace build was initiated by **{{.Labels.initiator}}**.',
```

Comment:
From the `Hi {{.UserName}},` to the next line would be nice to have two line breaks instead of one.

Reply:
Maybe adding the template version could be helpful.

Reply:
Good idea 👍 Fixed

Reply:
> Hm.. not sure about this... wouldn't it be too much information?

Hm... maybe? But since it is sent to admins I think it would be fine. See this conversation we had in the previous PR. https://github.com/coder/coder/pull/14262#discussion_r1717240037. 

It is not a blocker for me tho.

Reply:
Gotcha! Added 👍 Thank you for bringing it up.